### PR TITLE
Open polygon fix for Issue #23

### PR DESCRIPTION
**Pull request by mikest**
In this PR:
- Update for fixing #23, open polygons will no longer add arcs to beginning/end of lines.
- Added `.prettierrc` to match existing `.editorconfig` and updated all files to match formatting spec. (lots of quote updates.)
- Updated `package.json` a few dependencies which had security issues, some still remain as they're breaking changes.
- Update `.gitignore` to exclude `.DS_Store` :smile:
- Add `evt.metaKey` for toggling open/closed behavior in demo. Defaults to open-path, _you may or may not like this behavior._
- Bump version number to 0.1.0 -> 0.1.1
- Match license in `package.json` to MIT
- remove a spurious `console.log(...)`

image

image

### CODE REVIEW

**Comment by mikest** [comment]
Apologies for not splitting out the formatting from the code fix. If it's super irritating I can remove those and resubmit.

**Comment by BrunoFenzl** [comment]
Thanks for working on this @mikest! I skimmed through your changes and everything looks good! I'll pull your changes, do 
 some tests and get back to you. Thank you again.

File: demo/main.js [hunk 1]
```diff
@@ -119,12 +120,12 @@ class SVGPreview {
...
     if (evt.shiftKey) return;
 
-    const marker = this.commands.length ? "L" : "M";
+    const marker = this.commands.length ? 'L' : 'M';
     const x = evt.clientX - this.stageOffset.left;
     const y = evt.clientY - this.stageOffset.top;
     this.commands.push({ marker, values: { x, y } });
     this.dots.push(this.newDot(x, y));
-    this.updatePath();
+    this.updatePath(evt.metaKey);
```
Comment (BrunoFenzl) [inline demo/main.js:128]: Works great! Please, just add a short note in the controls box that click + (cmd/windows) keys closes the path.

**Review by BrunoFenzl — CHANGES_REQUESTED** [review]
Thank you for working on this! Everything works as expected. Please, just take a look in my comment.

File: demo/main.js (hunk 1 above)
Comment (mikest) [reply demo/main.js:128]: will do!

**Review by BrunoFenzl — APPROVED** [review]
Great! Everything looks good. I'll merge it. Thanks again for working on this :)

**Pull request merged**